Assets/Scripts/Layers/Properties/PropertyData/ILayerPropertyDataWithAssets.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
+
+namespace Netherlands3D.Twin.Layers.Properties
+{
+    public interface ILayerPropertyDataWithAssets
```

Comment:
Waarom is dit een Interface? is een AssetPropertyData : LayerPropertydata niet genoeg?

Reply:
Compositie vs inheritence; PropertyData met assets is maar een variant en door met interfaces te werken kan je propertydata samenstellen met meerdere varianten omdat je meerdere interfaces op 1 class kan zetten. Door met interfaces te werken kan je compositie verkrijgen en voorkom je problemen met kunstmatig grote of moeilijke inheritance bomen